samples/apps/copilot-chat-app/webapp/src/components/chat/ChatInput.tsx

```diff
@@ -58,14 +58,15 @@ export const ChatInput: React.FC<ChatInputProps> = (props) => {
     const [recognizer, setRecognizer] = React.useState<speechSdk.SpeechRecognizer>();
     const [isListening, setIsListening] = React.useState(false);
     const speechService = useSKSpeechService(process.env.REACT_APP_BACKEND_URI as string);
+    const bEnableSpeech = process.env.ENABLE_SPEECH_INPUT;
```

Comment:
rename to "enableSpeech"

Reply:
renamed to REACT_APP_ENABLE_SPEECH, the value isnt picked up correctly without that prefix.